SparkPost/2web2ui
# SA-1732: Fix Summary Report precision & date range with metrics rollup on initial load

### What Changed
 - Fixed precision being set to 1 min upon initial load in OG metrics rollup summary page.
 - Deleted unnecessary datepicker file.
 - Precision selector now shows correct selected precision. 

### How To Test
 - enable metrics-rollup and go to OG summary report page. Verify that it works.
 - Go to hibana report builder and open the date picker. Pick different date ranges and verify that the precision selector changes based on restricted/recommended precisions.
 - Go to Blocklist incidents collection and verify the date picker there still works.
